.github/workflows/test.yml

```diff
@@ -8,10 +8,10 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ruby-version: [2.5, 2.6, 2.7]
+        ruby-version: [2.5, 2.6, 2.7, '3.0', 3.1, 3.2]
```

Comment:
are the quotes around `3.0` really necessary?

Reply:
Yes - https://github.com/actions/runner/issues/849